package.json

```diff
@@ -89,9 +89,9 @@
     "@aws-sdk/client-sqs": "^3.352.0",
     "@babel/core": "^7.23.2",
     "@babel/eslint-parser": "^7.22.15",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
     "@babel/plugin-transform-class-properties": "^7.22.5",
     "@babel/plugin-transform-object-rest-spread": "^7.22.15",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
```

Comment:
The purpose of this package is to handle pre-building C++ plugins to the node env instance that an app is running on. The documentation of that package lists out requirements to use it. In those we see that we need to have `binaries` listed as an object in our `package.json` with a destination to compile the modules from (this can include an S3 bucket). We don't have that configuration in our package.json implying that this package is not being used. We may just be able to remove it